src/graphql/data/nft/Asset.ts

```diff
@@ -0,0 +1,176 @@
+import graphql from 'babel-plugin-relay/macro'
+import { parseEther } from 'ethers/lib/utils'
+import { GenieAsset } from 'nft/types'
+import { loadQuery, usePaginationFragment, usePreloadedQuery } from 'react-relay'
+
+import RelayEnvironment from '../RelayEnvironment'
+import { AssetPaginationQuery } from './__generated__/AssetPaginationQuery.graphql'
+import { AssetQuery, NftAssetsFilterInput, NftAssetSortableField } from './__generated__/AssetQuery.graphql'
+
+const assetPaginationQuery = graphql`
+  fragment AssetQuery_nftAssets on Query @refetchable(queryName: "AssetPaginationQuery") {
+    nftAssets(
+      address: $address
+      orderBy: $orderBy
+      asc: $asc
+      filter: $filter
+      first: $first
+      after: $after
+      last: $last
+      before: $before
+    ) @connection(key: "AssetQuery_nftAssets") {
+      edges {
+        node {
+          id
+          name
+          ownerAddress
+          image {
+            url
+          }
+          smallImage {
+            url
+          }
+          originalImage {
+            url
+          }
+          tokenId
+          description
+          animationUrl
+          suspiciousFlag
+          collection {
+            name
+            isVerified
+            image {
+              url
+            }
+            creator {
+              address
+              profileImage {
+                url
+              }
+              isVerified
+            }
+            nftContracts {
+              address
+              standard
+            }
+          }
+          listings(first: 1) {
```

Comment:
should this be configurable with a variable that defaults to 1?

Reply:
Currently we only ever want the cheapest (ie first) listing